### PR TITLE
Fixing and re-writing a broken notebook

### PR DESCRIPTION
**Pull request by xczuba**

### CODE REVIEW

File: recipes/counterfactual_data_augmentation.ipynb [hunk 1]
```diff
@@ -27,20 +28,21 @@
...
-   "execution_count": 7,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Authentication\n",
     "import os\n",
     "\n",
-    "# !pip install kili # uncomment if you don't have kili installed already\n",
+    "!pip install --upgrade kili # uncomment if you don't have kili installed already\n",
```
Comment (PierreLeveau) [inline recipes/counterfactual_data_augmentation.ipynb:38]:
```suggestion
    "!pip install --upgrade kili \n",
```

File: recipes/counterfactual_data_augmentation.ipynb [hunk 2]
```diff
@@ -27,20 +28,21 @@
...
     "from kili.client import Kili\n",
     "\n",
-    "api_endpoint = os.getenv('KILI_API_ENDPOINT') \n",
+    "# api_endpoint = os.getenv('KILI_API_ENDPOINT') \n",
     "# If you use Kili SaaS, use the url 'https://cloud.kili-technology.com/api/label/v2/graphql'\n",
+    "api_endpoint = \"https://cloud.kili-technology.com/api/label/v2/graphql\"\n",
     "\n",
-    "kili = Kili(api_endpoint=api_endpoint)\n",
+    "kili = Kili(api_endpoint=api_endpoint, api_key=\"6dc85cf3-3bbd-47db-bb26-1915f459489d\")\n",
```
Comment (PierreLeveau) [inline recipes/counterfactual_data_augmentation.ipynb:45]: no private key in Github. If this is an active one you should revoke it.

Comment (PierreLeveau) [reply recipes/counterfactual_data_augmentation.ipynb:45]: and it is assumed that the KILI_API_KEY is an environment variable

**Review by PierreLeveau — CHANGES_REQUESTED** [review]

File: recipes/counterfactual_data_augmentation.ipynb (hunk 2 above)
Comment (FlorianLeRoyKili) [reply recipes/counterfactual_data_augmentation.ipynb:45]: I think the best way, given the default behavior of Kili client would be: 
```
# Don't forget to set your 'KILI_API_KEY' environment variable with your API Key: os.environ['KILI_API_KEY'] = \"<YOUR_API_KEY>\"
# If you use Kili on-premise, be sure to set your 'KILI_API_ENDPOINT' environment variable.
kili = Kili()
```

File: recipes/counterfactual_data_augmentation.ipynb [hunk 3]
```diff
@@ -27,20 +28,21 @@
...
     "user_id = kili.auth.user_id"
```
Comment (FlorianLeRoyKili) [inline recipes/counterfactual_data_augmentation.ipynb:46]: delete line (user_id never used later)

**Review by FlorianLeRoyKili — CHANGES_REQUESTED** [review]

File: recipes/counterfactual_data_augmentation.ipynb (hunk 3 above)
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb]: Made the changes. Please have a look and resolve if poss.

File: recipes/counterfactual_data_augmentation.ipynb [hunk 4]
```diff
@@ -8,14 +8,15 @@
...
-    "2. We'll also try to [reproduce the results of the paper](##Reproducing-the-results), using similar models, to show how such a technique can be of key interest while working on a text-classification task.\n",
-    "We'll use the data of the study, both IMDB and Stanford NLI, publicly available [here](https://github.com/acmi-lab/counterfactually-augmented-data).\n",
+    "In this tutorial, we'll:\n",
+    "1. Create projects in Kili, both for [IMDB](##Data-Augmentation-on-IMDB-dataset) and [SNLI](##Data-Augmentation-on-SNLI-dataset) datasets, to reproduce such a data-augmentation task, in order to improve our model and decrease its variance when used in production with unseen data.\n",
+    "2. Try to [reproduce the results of the paper](##Reproducing-the-results), by using similar models, to show how such a technique can be of key interest while working on a text-classification task.\n",
```
[the remark this answers is not in the document]
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb]: Made the changes. Please have a look and resolve if poss.

File: recipes/counterfactual_data_augmentation.ipynb [hunk 5]
```diff
@@ -8,14 +8,15 @@
...
     "\n",
-    "We'll see in this tutorial :\n",
-    "1. How to create a project in Kili, both for [IMDB](##Data-Augmentation-on-IMDB-dataset) and [SNLI](##Data-Augmentation-on-SNLI-dataset) datasets, to reproduce such a data-augmentation task, in order to improve our model, and decrease its variance when used in production with unseen data.\n",
-    "2. We'll also try to [reproduce the results of the paper](##Reproducing-the-results), using similar models, to show how such a technique can be of key interest while working on a text-classification task.\n",
-    "We'll use the data of the study, both IMDB and Stanford NLI, publicly available [here](https://github.com/acmi-lab/counterfactually-augmented-data).\n",
+    "In this tutorial, we'll:\n",
+    "1. Create projects in Kili, both for [IMDB](##Data-Augmentation-on-IMDB-dataset) and [SNLI](##Data-Augmentation-on-SNLI-dataset) datasets, to reproduce such a data-augmentation task, in order to improve our model and decrease its variance when used in production with unseen data.\n",
```
[the remark this answers is not in the document]
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb]: Made the changes. Please have a look and resolve if poss.

File: recipes/counterfactual_data_augmentation.ipynb [hunk 6]
```diff
@@ -8,14 +8,15 @@
     "\n",
```
[the remark this answers is not in the document]
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb:18]: Made the changes. Please have a look and resolve if poss.

File: recipes/counterfactual_data_augmentation.ipynb (hunk 2 above)
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb]: Made the changes. Please have a look and resolve if poss.

File: recipes/counterfactual_data_augmentation.ipynb (hunk 1 above)
Comment (xczuba) [reply recipes/counterfactual_data_augmentation.ipynb]: Made the changes. Please have a look and resolve if poss.

**Review by PierreLeveau — APPROVED** [review]

**Pull request merged**